src/Utility/FieldFunctions.h

```diff
@@ -212,6 +212,10 @@ class FieldFuncStream : public RealFunc, private FieldFuncBase
 class FieldsFuncBase : public FieldFuncHDF5
 {
 protected:
+  //! \brief Construct from vector of fields.
+  //! \param fields Fields to use
+  FieldsFuncBase(const std::vector<Fields*> fields);
```

Comment:
Missing `&` in argument. Now it'll be pass-by-value. Was that intended here? At least it is inconsistent with the other constructors here.

Reply:
yeah, that's a whoopsie. should be passed by reference. of course it works fine as-is and copy is cheap but no reason to copy the vector either.